From f4faa06afbbbe0f7b2e9ee33a0a9afa8f818e788 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@gmail.com>
Date: Thu, 15 Apr 2021 11:28:05 -0400
Subject: [PATCH] cli: use external GO111MODULE and cross compiler

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 scripts/build/binary | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/scripts/build/binary b/scripts/build/binary
index 44cd4f14fd..039bd5ecfd 100755
--- a/scripts/build/binary
+++ b/scripts/build/binary
@@ -13,8 +13,6 @@ fi
 
 echo "Building $GO_LINKMODE $(basename "${TARGET}")"
 
-export GO111MODULE=auto
-
 if [ "$(go env GOOS)" = "windows" ]; then
   if [ ! -x "$(command -v goversioninfo)" ]; then
     >&2 echo "goversioninfo not found, skipping manifesting binary"
@@ -24,6 +22,6 @@ if [ "$(go env GOOS)" = "windows" ]; then
   fi
 fi
 
-(set -x ; go build -o "${TARGET}" -tags "${GO_BUILDTAGS}" -ldflags "${GO_LDFLAGS}" ${GO_BUILDMODE} "${SOURCE}")
+(set -x ; ${GO} build -trimpath -o "${TARGET}" -tags "${GO_BUILDTAGS}" -ldflags "${GO_LDFLAGS}" ${GO_BUILDMODE} "${SOURCE}")
 
 ln -sf "$(basename "${TARGET}")" "$(dirname "${TARGET}")/docker"
